config/routes.rb

```diff
@@ -114,6 +114,7 @@ def manage_matches
     end
   end
   resources :units, except: :show, concerns: [:restorable] do
+    post :deactivate
```

Comment:
do we need a restore route here too?

Reply:
We already had a restore route, that was previously used for Delete, so I reused that one instead